packages/context/SidebarProvider.tsx

```diff
@@ -87,27 +66,14 @@ export const SidebarContextProvider: React.FC = ({ children }) => {
 
     return dynamicAppsSelection;
   }, [availableApps, selectedApps]);
-  const { width: windowWidth } = useWindowDimensions();
-
-  useEffect(() => {
-    setIsSidebarExpanded(
-      windowWidth >= MOBILE_WIDTH ? defaultValue.isSidebarExpanded : false
```

Comment:
burger menu is now open on first load on small screens

Reply:
I think it make sense to have 2 state, the redux is used on large screen, but on mobile we use a context state, I don't think there is case where we want the menu to be open when we navigate to the site on mobile

Reply:
On load, I could set it to "undefined" by default and then create a condition where it will be closed for mobile breakpoints and open for other screens.
Then once opened or closed, it will follow whatever is on redux.

Reply:
this is nice, but do we really want to keep the menu open on mobile when we close and reopen the app/site ?

Reply:
I understand your point of view, but in order to maintain consistency, I saw we "remember" in both cases. (mobile and desktop).
What do you think?